css/pages/review.scss

```diff
@@ -0,0 +1,70 @@
+// Review
+.page {}
+
+.page--review {
+  color: #404040;
+
+  .page--review__title {
```

Comment:
Everything that's `.page--review` starting from here can be changed to `.review__title`.  `.page--review` should only be used for the parent above since you're defining a page, but here you're defining a title.